test/lib/mayaUsd/render/mayaToHydra/testTransforms.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Luma Pictures
```

Comment:
Did Luma really make a contribution to this file?  Also, the copyright date is wrong.